I think the build fail is real (though surprising). Maybe caching was added after the workaround PR?

```
======================================================================
ERROR: test_basic_fixture_generation (casexml.apps.phone.tests.test_ota_fixtures.OtaFixtureTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/corehq/util/test_utils.py", line 228, in inner
    return helper(*args, **kwargs)
  File "/mnt/corehq/util/test_utils.py", line 201, in __call__
    call_with_settings(fn_with_pre_and_post, run_config.settings, args, kwargs)
  File "/mnt/corehq/util/test_utils.py", line 216, in call_with_settings
    fn(*args, **kwargs)
  File "/mnt/corehq/util/test_utils.py", line 197, in fn_with_pre_and_post
    self.fn(*args, **kwargs)
  File "corehq/ex-submodules/casexml/apps/phone/tests/test_ota_fixtures.py", line 80, in test_basic_fixture_generation
    fixture_xml = generator.get_fixtures(self.casexml_user, version=V2)
  File "corehq/ex-submodules/casexml/apps/phone/fixtures.py", line 90, in get_fixtures
    return self._get_fixtures(group, None, user, version, last_sync, app)
  File "corehq/ex-submodules/casexml/apps/phone/fixtures.py", line 75, in _get_fixtures
    for provider in providers])
  File "/mnt/corehq/apps/fixtures/fixturegenerators.py", line 54, in __call__
    all_types = dict([(t._id, t) for t in FixtureDataType.by_domain(user.domain)])
  File "/mnt/corehq/apps/fixtures/models.py", line 68, in by_domain
    return get_fixture_data_types_in_domain(domain)
  File "/mnt/corehq/util/quickcache.py", line 308, in inner
    return helper(*args, **kwargs)
  File "/mnt/corehq/util/quickcache.py", line 98, in __call__
    self.cache.set(key, content)
  File "/mnt/corehq/util/quickcache.py", line 55, in set
    cache.set(key, value)
  File "/mnt/corehq/util/quickcache.py", line 19, in set
    return self.cache.set(key, value, timeout=self.timeout)
  File "/vendor/lib/python2.7/site-packages/django/core/cache/backends/locmem.py", line 67, in set
    pickled = pickle.dumps(value, pickle.HIGHEST_PROTOCOL)
PicklingError: Can't pickle <type 'function'>: attribute lookup __builtin__.function failed
----------------------------------------------------------------------
```

--- a/corehq/apps/fixtures/dbaccessors.py
+++ b/corehq/apps/fixtures/dbaccessors.py
@@ -16,14 +16,14 @@ def get_number_of_fixture_data_types_in_domain(domain):
 @quickcache(['domain'], timeout=30 * 60)
 def get_fixture_data_types_in_domain(domain):
     from corehq.apps.fixtures.models import FixtureDataType
-    return FixtureDataType.view(
+    return list(FixtureDataType.view(
         'by_domain_doc_type_date/view',
         endkey=[domain, 'FixtureDataType'],
         startkey=[domain, 'FixtureDataType', {}],
         reduce=False,
         include_docs=True,
         descending=True,
-    )
+    ))
 
 
 def get_owner_ids_by_type(domain, owner_type, data_item_id):

